.github/workflows/build_test.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nosetests
+on: [push]
+
+jobs:
+  build:
+    name: Validate Changes
+    runs-on: ubuntu-latest
```

Comment:
Is it worth pinning this to ubuntu-18.04? I don't like surprises.

Reply:
Yeah, we might as well be explicit.